DigitalLearningSolutions.Data.Tests/Services/LearningHubAuthApiUrlServiceTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using System;
+    using System.Web;
+    using DigitalLearningSolutions.Data.Helpers;
+    using DigitalLearningSolutions.Data.Services;
+    using FakeItEasy;
+    using FluentAssertions;
+    using Microsoft.Extensions.Configuration;
+    using NUnit.Framework;
+
+    public class LearningHubAuthApiUrlServiceTests
+    {
+        private LearningHubAuthApiUrlService service = null!;

```

Comment:
Could we change this service name to something more meaningful? e.g. `learningHubAuthApiUrlService`